docs/hub-tutorials/join-testnet.md

```diff
@@ -1,29 +1,29 @@
 <!--
-order: 4
+order: 3
 -->
 
 # Join the Public Testnet
 
 | Current Upgrade | Chain Id       | Upgrade Block Height | Upgrade Date     |
 | --------------- | -------------- | -------------------- | ---------------- |
-| Theta           | `theta-testnet-001` | 9283650   | March 17 2021 |
+| Theta           | `theta-testnet-001` | TBD   | March 17 2021 |
 
 
 ## Background
-The current Cosmos Hub Testnet is running on the [Theta Upgrade](https://interchain-io.medium.com/preparing-for-the-cosmos-hub-v7-theta-upgrade-2fc41ce34787). Visit the [testnet explorer](https://explorer.theta-testnet.polypore.xyz/) to view all on chain activity.
+The current Cosmos Hub Testnet is running to prepare for the [Theta Upgrade](https://interchain-io.medium.com/preparing-for-the-cosmos-hub-v7-theta-upgrade-2fc41ce34787). Visit the [testnet explorer](https://explorer.theta-testnet.polypore.xyz/) to view all on chain activity.
```

Comment:
The theta upgrade already happened...do you mean Rho?

Reply:
```suggestion
The current Cosmos Hub Testnet is running on the [Theta Upgrade](https://interchain-io.medium.com/preparing-for-the-cosmos-hub-v7-theta-upgrade-2fc41ce34787). Visit the [testnet explorer](https://explorer.theta-testnet.polypore.xyz/) to view all on chain activity.
```